configure

```diff
@@ -164,7 +164,7 @@ if ! grep $pat $srcdir/configure > /dev/null 2>&1; then
   exit 1
 fi
 
-if ! `cmp --help > /dev/null 2>&1`; then
+if ! `command -v cmp > /dev/null 2>&1`; then
   echo "Sorry, you need 'cmp' to configure GHDL.";
```

Comment:
I have in my local tree a fix for that.  It just test cmp between /dev/null and /dev/null, instead of relying on `command`.